otherPages/type/typeDrink.html

```diff
@@ -42,6 +42,35 @@
             
     </section>
     <!-- End Cards Section -->
+    <!-- Recipe popup -->
+    <div class="overlay">
+        <div class="pop-up"> 
+            <div id="cont">
+            <i class="fas fa-times-circle close"></i>
```

Comment:
add the functionality of it

Reply:
It's a container for the head of the pop-up, I changed the id name